sys/include/net/gnrc/netif.h

```diff
@@ -300,6 +300,33 @@ struct gnrc_netif_ops {
     void (*msg_handler)(gnrc_netif_t *netif, msg_t *msg);
 };
 
+/**
+ * @brief Context of GNRC Netif event based Bottom Half Processor
+ */
+typedef struct gnrc_netif_bhp_ctx {
+    event_t event;          /**< Event to be called on BH */
```

Comment:
since the radio just signals the existence of an IRQ, I think it would be hard to correlate back the order of events. The driver was also designed with this in mind.